week_six/quiz_class.dart

```diff
@@ -14,7 +14,9 @@ void main() {
 }
 
 List<int> findPair(List<int> numbers, int target) {
-  // Your code here
-
+  if (numbers.length == target) {
+    int result = numbers.length;
+    return [result];
+  }
```

Comment:
@Muhammad-Junaid95 where is the code?

Reply:
Sir it's only showing the changes I made